Prox/Prox/PlaceDetails/FilterViewController.swift

```diff
@@ -18,14 +18,19 @@ class FilterViewController: UIViewController {
 
     fileprivate let background = UIButton()
     private let stackView = UIStackView()
-    private let filters: [PlaceFilter]
-    private(set) var enabledFilters: [Bool]
+    private(set) var enabledFilters: Set<Filter>
     private let placeCountLabel = UILabel()
     private let topRatedSwitch = UISwitch()
 
-    init(filters: [PlaceFilter], enabledFilters: [Bool], topRatedOnly: Bool) {
-        self.filters = filters
-        self.enabledFilters = enabledFilters
+    private let filterLabels: [Filter: String] = [
```

Comment:
Thanks for mentioning ordering -- I actually had this as an array of (enum, label) tuples before this PR, then "cleaned" it up to be a dictionary. But your comment reminds me of why I didn't use a dictionary in the first place: we lose the ordering! So I'll probably go back to tuples here.

My reasoning for keeping this in FVC is that strings are user-facing, so I prefer the presentation layer to own them rather than mixing them with the model.